### PR TITLE
Adding new interconnectors

### PR DESCRIPTION
**Pull request by Charlottevm**
New interconnectors (7-12) have been added to the front-end. 
This enables countries with more than 6 interconnectors (such as Germany) to model their interdependencies to other countries.

### CODE REVIEW

**Comment by Charlottevm** [comment]
The interconnectors in the price curve upload function are not correctly yet (from 1-12) but this will be done during the next dev. sprint.

File: app/views/scenarios/slides/_interconnector_curves_upload.html.haml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.interface_group
+  %hr
+  %span.label= t("custom_curves.interface_group_title")
+
+%p{ style: 'margin-top: 0' }
+  = t("custom_curves.interconnector_intro_html", docs_path: 'https://docs.energytransitionmodel.com/main/costs-imported-electricity#uploaded-price-curves')
```
Comment (mabijkerk) [inline app/views/scenarios/slides/_interconnector_curves_upload.html.haml:6]: @antw is it possible to make this link open in a new tab? When I click it now I'm immediately referred to the documentation, thereby leaving the model, while we normally let external links in the model open in a new tab.

Comment (antw) [reply app/views/scenarios/slides/_interconnector_curves_upload.html.haml:6]: Done.

**Comment by mabijkerk** [comment]
On hold until the addition of the curve download functionality has been approved by the client.

**Pull request merged**